SECURITY.md

```diff
@@ -0,0 +1,18 @@
+# Security Policy
+
+if you believe you have found a security vulnerability in our repository, please report it to us as described below.
+
+## Reporting a Vulnerability
+
+To report a vulnerability, send and email to the adress below : [dsi-oscoe-enedis@enedis.fr](mailto:dsi-oscoe-enedis@enedis.fr)
```

Comment:
`Instead, please send an email to [dsi-oscoe-enedis@enedis.fr](mailto:dsi-oscoe-enedis@enedis.fr)
`

Beaucoup plus clair et sans faute d'anglais

Reply:
De plus, à mon sens la phrase en gras `Please do not report security vulnerabilities through public GitHub issues, discussions, or pull requests.` est importante à conserver